apps/financial-aid/backend/src/app/modules/application/application.controller.ts

```diff
@@ -209,11 +209,13 @@ export class ApplicationController {
   })
   async createEvent(
     @Body() applicationEvent: CreateApplicationEventDto,
+    @CurrentUser() user: User,
   ): Promise<ApplicationModel> {
     await this.applicationEventService.create(applicationEvent)
 
     const application = await this.applicationService.findById(
       applicationEvent.applicationId,
+      user,
```

Comment:
Should findById take just service since it is not using anything else from user?